var/spack/repos/builtin/packages/melissa-api/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class MelissaApi(CMakePackage):
+    """Melissa is a file-avoiding, adaptive, fault-tolerant and elastic
+    framework, to run large-scale sensitivity analysis or deep-surrogate
+    training on supercomputers.
+    This package builds the API used when instrumenting the clients.
+    """
+
+    homepage = "https://gitlab.inria.fr/melissa/melissa"
+    # url = <url-to-archive>
+    git = "https://gitlab.inria.fr/melissa/melissa.git"
+    maintainers = ["robcaulk", "mschouler", "raffino"]
```

Comment:
Please use the new `maintainers` syntax/directive here and in `py-meliisa`.  See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers.

```suggestion
    maintainers("robcaulk", "mschouler", "raffino")
```